platforms/software/build-init/src/test/groovy/org/gradle/buildinit/plugins/internal/BuildScriptBuilderGroovyTest.groovy

```diff
@@ -176,8 +178,8 @@ repositories {
 dependencies {
     // Use slf4j
     implementation 'org.slf4j:slf4j-api:2.7'
-    implementation 'org.slf4j:slf4j-simple:2.7'
 
+    implementation 'org.slf4j:slf4j-simple:2.7'
```

Comment:
❌ This new line is caused by not having a way to group dependencies with a comment anymore. This should be fixed in order to produce more readable scripts.

Reply:
Ability to group dependencies with a comment is restored